Java/core/src/main/java/com/amazon/randomcutforest/sampler/CompactSampler.java

```diff
@@ -542,4 +370,24 @@ public CompactSampler(Builder<?> builder) {
             }
         }
     }
+
+    public CompactSampler(int sampleSize, double lambda, Random random, boolean storeSequences) {
```

Comment:
Did you mean to delete the old constructors?

Reply:
yes, but then the number of files became too large and I did not want to change the tests simultaneously. Lets have a separate PR for removing these constructors in the tests (and in the main forest class).